MovieMVP/MovieMVP/MoviesTableViewModule/Model/Movie.swift

```diff
@@ -0,0 +1,38 @@
+// Movie.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftyJSON
+
+///  Фильм
+struct Movie {
+    /// adult - Кино для взрослых?
```

Comment:
```suggestion
    /// Кино для взрослых?
```
Удали дубли везде и так понятно по названию свойства